hx_lti_initializer/views.py

```diff
@@ -369,7 +371,8 @@ def course_admin_hub(request):
             'debug': debug,
             'starter_object': starter_object,
             'starter_object_id': object_id,
-            'starter_collection_id': collection_id
+            'starter_collection_id': collection_id,
+            'utm_source': request.session.session_key if not request.session['is_staff'] else '',
```

Comment:
You're right. I think monkey-patching those functions might be the right way to go. We just needed a quick solution to this for a course we launched a bit ago. Should we merge and then fix or wait for the more elegant solution?